torch/export/unflatten.py

```diff
@@ -1423,79 +1430,42 @@ class _IVals:
     Here ival1 and ival2 are intermediate values created inside
     calls to n0 and n1 respectively, and used inside calls to
     n1 and n2 respectively.
-
-    Thus our analysis will produce {ival1: {n0, n1}, ival2: {n1, n2}}.
     """
 
     def __init__(self):
-        # ival node name -> set of fqns that create and use it
-        self.fqns = defaultdict(set)
-        # ival node name -> tensor storage for corresponding attribute
-        self.storage = {}
-
-    def read(self, fqn, graph, node):
-        """
-        Read attribute corresponding to a given intermediate value.
-        """
-        # to read ival x, get attribute __ival__x
-        with graph.inserting_before(None):
-            ival_node = graph.get_attr("__ival__" + node.name, type_expr=node.type)
-            ival_node.meta = copy.copy(node.meta)
-
-        if node.name not in self.storage:
-            # create empty tensor matching fake, using a cache
-            # to ensure the same tensor is returned per ival_name
-            fake = node.meta["val"]
-            self.storage[node.name] = torch.empty(fake.shape, dtype=fake.dtype)
-        self.fqns[node.name].add(fqn)
+        # set of node names corresponding to intermediate values
+        self.node_names = set()
 
-        return ival_node
+    def _is_mutable(self, target):
```

Comment:
There are some ops that are maybe_aliasing or aliasing that you should extra check.

Reply:
I checked that `.to` is still a mutation. Test coming up...